src/app/containers/SwapFormContainer/index.tsx

```diff
@@ -313,6 +317,18 @@ export const SwapFormContainer: React.FC = () => {
                 <AssetRenderer asset={targetToken} />
               </div>
             </div>
+
+            <div className={styles.swapBtnHelper}>
+              <div>{t(translations.swap.maximumPrice)}</div>
+              <div>
+                <AssetValue
+                  value={maximumPrice}
```

Comment:
Price is shown as 0 in some cases:
![image](https://user-images.githubusercontent.com/765825/219743908-8bb427d2-1eb0-4fa6-95f1-10a0f006d618.png)

And I think price shown should be RBTC/XUSD price (e.g. $20,000) instead of XUSD/RBTC price (1.01 XUSD). 
![image](https://user-images.githubusercontent.com/765825/219745862-5ee20991-b141-4e0e-ac05-f2a7dd9160a5.png)

Haven't tested other pairs yet but I assume issue also applies elsewhere. We should already have logic in a hook/util function somewhere to help format this, used for showing entry prices etc, as it's different depending which pair is being displayed.

Reply:
The issue you experienced is based on the fact that `RBTC` and `BNB` on testnet has value of `0 USD` (you can see it on `Portfolio` page as well). The behaviour is not present on mainnet so try to test it there.

I'll take a look at the other issue.